pyomo/common/unittest.py

```diff
@@ -0,0 +1,152 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import math
+import six
+
+# Import base classes privately (so that we hae handles on them)
+import pyutilib.th.pyunit as _pyunit
+from pyutilib.th.pyunit import unittest as _unittest
+
+# Now, import the base unittest environment.  We will override things
+# specifically later
+from unittest import *
+from pyutilib.th import *
+
+from pyomo.common.collections import Mapping, Sequence
+
+class TestCase(_pyunit.TestCase):
+    """A Pyomo-specific class whose instances are single test cases.
+
+    This class derives from unittest.TestCase and provides the following
+    additional functionality:
+      - extended suport for test categories
+      - additional assertions:
+        * :py:meth:`assertRelativeEqual`
```

Comment:
I don't know if `meth` is a good word choice.

Reply:
Not our choice: this is a Sphinx RST flag: see the Sphinx [docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#role-py-meth)